lib/CGameInfoCallback.cpp

```diff
@@ -781,12 +781,12 @@ int CPlayerSpecificInfoCallback::getHeroSerial(const CGHeroInstance * hero, bool
 	size_t index = 0;
 	auto & heroes = gs->players[*getPlayerID()].heroes;
 
-	for (auto & heroe : heroes)
+	for (auto & hero : heroes)
 	{
-		if (includeGarrisoned || !(heroe)->inTownGarrison)
+		if (includeGarrisoned || !(hero)->inTownGarrison)
 			index++;
 
-		if (heroe == hero)
+		if (hero == hero)
```

Comment:
Something went wrong here.

Reply:
Fixed in https://github.com/vcmi/vcmi/pull/4195